react-main/src/scss/index.scss

```diff
@@ -41,3 +41,22 @@
 .game-info {
 	margin-left: 20px;
 }
+
+.formContainer {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.inputContainer {
```

Comment:
The styles of .formContainer are also contained by .inputContainer 
You can define those styles without repeating them for each class.